sdk/rust/build.rs

```diff
@@ -17,7 +17,7 @@ fn main() {
     let commit = if output.status.success() {
         String::from_utf8(output.stdout).unwrap()
     } else {
```

Comment:
Can't comment on the correct line, but if git isn't installed I think line 15 will fail if `git` isn't available in your PATH